Bai-tap-Function-2/Bai5.py

```diff
@@ -0,0 +1,34 @@
+'''
+Bài 5: Viết hàm nhận vào n (n là cạnh của tam giác cân) => In ra hình tam giác cân như Ví dụ.
+
+Input: func5(5)
+Output:
+____*
+___***
+__*****
+-*******
+*********
+'''
+
+def func5(n):
+  """
+  In ra hình tam giác cân có cạnh n
+
+  Args:
+    n: Độ dài cạnh của tam giác cân
+
+  Returns:
+    None
+  """
+
+  for i in range(n):
+    # In khoảng trắng
+    for j in range(n-i-1):
+      print(" ", end="")
+    # In dấu sao
+    for j in range(2*i+1):
+      print("*", end="")
+    print()
+
+# Gọi hàm với n = 5
+func5(8)
```

Comment:
@ngochan21098 Please add a new line.

Reply:
@ddhienpy I have fixed it here
https://github.com/ngochan21098/learning-python-basic/commit/4f0786b56802b21ec8c79670e2c95a79be05bde1